nautobot/ipam/templates/ipam/ipaddress_merge.html

```diff
@@ -0,0 +1,115 @@
+                                <input checked="true" type="hidden" id="address" name="address" value="{{ip.address}}">
+                                <label for="address">{{ip.address}}</label>
+                            </td>
+                            <td>
+                                <input type="radio" id="namespace" name="namespace" value="{{ip.parent.namespace.pk}}" required>
+                                <label for="namespace">{{ip.parent.namespace | placeholder}}</label>
+                            </td>
+                            <td>
+                                <input type="radio" id="mask_length" name="mask_length" value="{{ip.mask_length}}" required>
+                                <label for="mask_length">{{ip.mask_length | placeholder }}</label>
+                            </td>
+                            <td>
+                                <input type="radio" id="tenant" name="tenant" value="{{ip.tenant.pk}}" required>
+                                <label for="tenant">{{ip.tenant | placeholder}}</label>
+                            </td>
+                            <td>
+                                <input type="radio" id="status" name="status" value="{{ip.status.pk}}" required>
+                                <label for="status">{{ip.status | placeholder}}</label>
+                            </td>
+                            <td>
+                                <input type="radio" id="role" name="role" value="{{ip.role.pk}}" required>
+                                <label for="role">{{ip.role | placeholder}}</label>
+                            </td>
+                            <td>
+                                <input type="radio" id="description" name="description" value="{{ip.description}}" required>
+                                <label for="description">{{ip.description | placeholder }}</label>
+                            </td>
+                            <td>
+                                <input type="radio" id="tags" name="tags" value="{{ip.tags.all | get_tags_uuids}}">
+                                <label for="tags">
+                                    {% for tag in ip.tags.all %}
+                                        {% tag tag url %}
+                                    {% empty %}
+                                        <span class="text-muted">No tags assigned</span>
+                                    {% endfor %}
+                                </label>
+                            </td>
+                        </tr>
+                {% endfor %}
+                </tbody>    
+            </table>
+        </div>
+    </div>
+</div>
+<div class="row">
+    <div class="col-md-10 col-md-offset-9">
+        {% block buttons %}
+            <button type="submit" name="_merge" class="btn btn-primary">Merge and Go to the Next Duplicates</button>
+            <a href="{{ return_url }}" class="btn btn-default">Cancel</a>
+        {% endblock %}
+    </div>
```

Comment:
Probably would be good to have a "cancel" button that returns you to the IP list view?

Reply:
I do have that at line 110. It is a hyperlink disguised as a button.

Reply:
Reading comprehension fail on my part. Thanks. :)